### DISCUSSION TITLE
node collector in Talos - /etc/systemd read-only file system

### DISCUSSION THREAD

**Discussion by andyrue** [aquasecurity/trivy-operator#1326] [Help & Support] [answered]
I'm trying to get things set up on a Talos install which has most of /etc set as read-only, so I assume this is the reason for this, but wanted to make sure I was chasing the right thing before asking for support in Talos.

node collector is showing the following error.
failed to mkdir "/etc/systemd": mkdir /etc/systemd: read-only file system

**Comment by chen-keinan** [comment, marked as answer]
@andyrue you can override (exclude) volumeMounts/volumes if needed with this [flag](https://github.com/aquasecurity/trivy-operator/blob/main/deploy/helm/values.yaml#L508)

**Comment by skillpoint-dev** [comment]
Anyone finding this and wondering why you should use the "existingSecret" value - thats because the link references main which is of course changing over time...

I am pretty sure @chen-keinan meant to link to [this](https://github.com/aquasecurity/trivy-operator/blob/ce828a86a85ffa3abce8538d74ba2d72a9a7ed05/deploy/helm/values.yaml#L508)
